aep/general/0121/aep.md.j2

```diff
@@ -83,16 +83,19 @@ patterns, such as database transactions, import and export, or data analysis.
 
 ### Strong Consistency
 
-For methods that operate on the [management plane][], the completion of those
```

Comment:
While the notion of management plane cannot be precisely defined, it is indeed widely used in the industry. If there is no common guideline, each company will define their internal guide for it, that would lead to inconsistency when different companies work with each other.

For example, almost all Azure management APIs are exposed by Azure Resource Manager. It is a very successful design pattern. We will not force every company to adopt the model, but document it as a design pattern is fine IMO.

Reply:
Sure, I think it would be valuable to provide a definition - I did import the definition of such planes from AIPs: https://github.com/aep-dev/aep.dev/pull/85. 

But I don't think we should base a requirement on unclear definitions, so I think this PR is still valuable (making guidance work without relying on those).

Reply:
will follow up in #85